### ISSUE TITLE
IDE0060 shouldn't warn for throw-only methods

### ISSUE THREAD

**Issue by stephentoub** [dotnet/roslyn#65275]
IDE0060 currently fires for methods like:
```C#
private static void Blah(int something) => throw new PlatformNotSupportedException();
```
We have many such constructs, due to multitargeting and having the same method in multiple files, in some fully implemented and in others just throwing an exception.  We're having to suppress IDE0060 in many places to account for this.  It'd be really helpful if either always or configurably IDE0060 could support not warning in cases where the method implementation is simply to always throw.

**Comment by sharwell** [comment]
@CyrusNajmabadi please update the **Feature - IDE0060** label to include the diagnostic title.

**Comment by CyrusNajmabadi** [comment]
Done.

**Comment by CyrusNajmabadi** [comment]
I'll try to slowly update these all accordingly.
